public/js/src/constants.ts

```diff
@@ -27,39 +27,40 @@ export const ARROW_COLOR = {
 export const FADE_TIME = 350; // In milliseconds
 
 // These constants much match their server-side counterparts
-export const ACTION = {
-  PLAY: 0,
-  DISCARD: 1,
-  COLOR_CLUE: 2,
-  RANK_CLUE: 3,
-};
-export const CLUE_TYPE = {
-  COLOR: 0,
-  RANK: 1,
-};
-export const REPLAY_ACTION_TYPE = {
-  TURN: 0,
-  ARROW: 1,
-  SOUND: 2,
-  HYPO_START: 3,
-  HYPO_END: 4,
-  HYPO_ACTION: 5,
-  HYPO_BACK: 6,
-  HYPO_TOGGLE_REVEALED: 7,
-};
-export const REPLAY_ARROW_ORDER = {
-  DECK: -1,
-  CLUES: -2,
-  PACE: -3,
-  EFFICIENCY: -4,
-  MIN_EFFICIENCY: -5,
-};
-export const STACK_DIRECTION = { // Used in the "Up or Down" and "Reversed" variants
-  UNDECIDED: 0,
-  UP: 1,
-  DOWN: 2,
-  FINISHED: 3,
-};
+export enum ActionType {
```

Comment:
airbnb guide says that constants should be uppercase with underscores

Reply:
This is not a constant anymore. It's a type now

Reply:
https://basarat.gitbook.io/typescript/styleguide#enum